src/main/java/com/newrelic/agent/instrumentation/verify/VerificationPlugin.java

```diff
@@ -39,7 +39,7 @@ public void apply(Project target) {
                 verifyInstrumentation,
                 verifyCapstoneTask,
                 target.getLogger(),
-                destinationDir));
+                destinationDir, new ProjectTaskFactory(target, target.getLogger(), destinationDir)));
```

Comment:
```suggestion
                destinationDir, 
                new ProjectTaskFactory(target, target.getLogger(), destinationDir)));
```

Consistency in formatting